builder/static_code.go

```diff
@@ -722,6 +723,17 @@ type Cloner interface {
 	Clone() interface{}
 }
 
+var statePool = &sync.Pool{
+	New: func() interface{} { return make(storeDict, 2) },
```

Comment:
Why is the `storeDict` initialized with a size of `2`? is there a particular reason for this?

Reply:
Good catch, this was an optimization attempt that is not performant.  Fixed in 2ee1908.